meta-balena-common/recipes-support/resin-mounts/resin-mounts.inc

```diff
@@ -72,6 +73,11 @@ do_compile () {
 			# sure that is mounted as well
 			sed -i -e "/^Requires=/s/\$/ bind-usr-share-ca-certificates-balena.service/" "$servicefile"
 			sed -i -e "/^After=/s/\$/ bind-usr-share-ca-certificates-balena.service/" "$servicefile"
+		elif [ "$bindmount" = "/etc/fake-hwclock" ]; then
+			# This bind mount needs to be running before the fake-hwclock service starts
+			sed -i -e "s/^\(Requires=\).*/\1resin-state.service resin-state-reset.service/" "$servicefile"
+			sed -i -e "s/^\(After=\).*/\1resin-state.service resin-state-reset.service/" "$servicefile"
```

Comment:
If I read this correctly it drops `var-volatile-lib.service` and `systemd-tmpfiles-setup.service` from both `Requires=` and `After=` is there any particular reason for that?

Reply:
We need `fake-hwclock` to run as early as possible in the boot process, so any unnecessary dependencies are dropped. The `bind-etc-fake-hwclock.service` only requires `resin-state.service` and `resin-state-reset.service`